app/models/course/lesson_plan/item.rb

```diff
@@ -40,6 +40,17 @@ def copy_attributes(other, time_shift)
     self.end_at = other.end_at + time_shift if other.end_at
   end
 
+  # Test if the lesson plan item has started for self directed learning.
+  #
+  # @return [Boolean]
+  def self_directed_started?
```

Comment:
I was intended to do that also. but you know, each model has their own `started?` ( we have an extension for doing this),  so override `lesson_plan` won't work for assessment/video/survey/etc, as in they will still call their own versions.